wiki/Contests/CCC/2024/en.md

```diff
@@ -0,0 +1,96 @@
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") 2nd place | 188 CNY |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") 3rd place | 88 CNY |
+| Participation Award (4th-7th) | 18 CNY |
+
+![CCC2024 badge](img/winner_badge.png)
+
+## Organisation
+
+| Position | Member(s) |
+| :-- | :-- |
+| Organiser | ::{ flag=CN }:: [_Stan](https://osu.ppy.sh/users/1653229) |
+| Manager | ::{ flag=CN }:: [_Stan](https://osu.ppy.sh/users/1653229), ::{ flag=CN }:: [mafumafu](https://osu.ppy.sh/users/3076909) |
+| Designer | ::{ flag=CN }:: [AlexDunk](https://osu.ppy.sh/users/9194799) |
+| Judge | ::{ flag=MY }:: [_Kobii](https://osu.ppy.sh/users/6209713), ::{ flag=SG }:: [Evening](https://osu.ppy.sh/users/2193881), ::{ flag=CN }:: [gzdongsheng](https://osu.ppy.sh/users/8660315), ::{ flag=ID }:: [Maxus](https://osu.ppy.sh/users/4335785), ::{ flag=CN }:: [Unpredictable](https://osu.ppy.sh/users/7560872) |
+| Statistician | ::{ flag=CN }:: [[Crz]Alleyne](https://osu.ppy.sh/users/11279273) |
+
+## Links
+
+- [Forum thread](https://osu.ppy.sh/community/forums/topics/1928815)
+- [Results Sheet](https://mappersguild.com/contests/results?contest=66026a8aa8c1495bcfbd23ea)
+- [Results Video](https://www.bilibili.com/video/BV1ZFv4enEiY/)
+- [Entries](https://drive.google.com/file/d/1IKg5NHqRxiqNuqAFSiq8rDMO-ADarGZX/view?usp=sharing)
+
+## Song List
+
+| Song | Length | BPM |
+| :-- | :-- | :-- |
+| you - Texture Garden | 02:28 | 128 |
+| Kirara Magic & Ulchero - Online Fantasy (feat. Moon Jelly) | 03:45 | 126 |
+| Essbee - Rebel Of Wildernes | 02:28 | 166 |
+| mybear - Last Slurp | 03:10 | 126 |
+| bobawhale & glitchless - starsignal! | 02:52 | 170 |
+| awfuless - Temptation | 03:04 | 173 |
+
+## Results
+
+| Place | Mapper(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") Winner | ::{ flag=CN }:: [BKwind](https://osu.ppy.sh/users/8900975) |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") 2nd place | ::{ flag=CN }:: [\_Yiiiii](https://osu.ppy.sh/users/6066359) |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") 3rd place | ::{ flag=CN }:: [tyrcs](https://osu.ppy.sh/users/13026904) |
+| Participation Award (4th-7th) | ::{ flag=CN }:: [Boomushroom](https://osu.ppy.sh/users/10588550), ::{ flag=CN }:: [Seiran-](https://osu.ppy.sh/users/14351534), ::{ flag=RU }:: [FlappiBakuse](https://osu.ppy.sh/users/11724295), ::{ flag=CN }:: [wIntEr7z](https://osu.ppy.sh/users/16509588) |
+
+## Ruleset
+
+### Contest rules
+
+- **This contest is limited to the `osu!mania` mode only.**
+- **This contest is exclusively for Chinese players, including those with the flags of ::{ flag=CN }:: Chinese mainland, ::{ flag=HK }:: Hong Kong, ::{ flag=MO }:: Macao, and ::{ flag=TW }:: Taiwan.** If a player's flag is not one of the four listed but they are still eligible, participation is allowed with proof of eligibility.
+- **Only account valid for logging into the [Mappers' Guild](https://mappersguild.com) are allowed.**
```

Comment:
since the submission need to be submitted by mg website, so if the account can't log in mg then the player can't participate the contest, that's what I mean and make a information like that.